space.crankshaft.Crankshaft.yml

```diff
@@ -0,0 +1,69 @@
+app-id: space.crankshaft.Crankshaft
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+  - org.freedesktop.Sdk.Extension.node18
+
+command: crankshaft
+
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
Used by application to check for running Steam processes, used by plugins for whatever they want (start an SSH server, control media, etc)